modules/nf-core/mageck/test/main.nf

```diff
@@ -14,7 +14,6 @@ process MAGECK_TEST {
     tuple val(meta), path("*.gene_summary.txt")  , emit: gene_summary
     tuple val(meta), path("*.sgrna_summary.txt") , emit: sgrna_summary
     tuple val(meta), path("*.R")                 , emit: r_script
-    tuple val(meta), path("*.Rmd")               , emit: r_report
```

Comment:
Why remove that here? If you dont want to use it downstream just keep it here but ignore it? Otherwise "optional: true" might also do it?

Reply:
It's actually no longer generated in that version i think it leads to the module failing